packages/components/src/components/shared/FormText.tsx

```diff
@@ -59,7 +32,8 @@ export const Header: FC<HeaderProps> = ({ text, required }) => {
   if (!text) return null
 
   const textStyle = {
-    ...fontHeader,
+    ...typography.vadsFontBodyLarge,
+    marginBottom: 0,
     color: theme.vadsColorForegroundDefault,
```

Comment:
Can't this be handled by our Text component with default tone and none bottomSpacing?

Reply:
Oh, or can it not because the `role` of heading while being body text?

Reply:
Yup. This was done due to the `role`